lib/tlog/play_conf.c

```diff
@@ -75,6 +75,9 @@ tlog_play_conf_load(struct tlog_errs **perrs,
     assert(pcmd_help != NULL);
     assert(pconf != NULL);
     assert(argv != NULL);
+    /* Check validate the config */
+    assert(tlog_play_conf_validate(NULL, conf, TLOG_CONF_ORIGIN_ARGS) ==
+            TLOG_RC_OK);
```

Comment:
This wouldn't make sense, we don't need to validate a NULL pointer we just initialized. This function doesn't need to validate the configuration it receives, as it doesn't receive any. The same for the rest of those.